src/services/findAllReferences.ts

```diff
@@ -1512,6 +1512,11 @@ namespace ts.FindAllReferences {
                     Debug.assertNever(state.specialSearchKind);
             }
 
+            // Use the parent symbol if the location is commonjs require syntax on javascript files only.
+            referenceSymbol = isInJSFile(referenceLocation) && isRequireVariableDeclaration(referenceLocation.parent)

```

Comment:
You should also check the `kind` of `referenceLocation` here, since the `require("whatever")` side’s parent is _also_ the `RequireVariableDeclaration`, and what you’re trying to ask here is if you have the left-hand side.

Reply:
I have changed it to check if the parent of `referenceLocation` is `BindingElement`. I think this is more accurate if we want to make sure we're checking on the `identifier`.